[2018-11-10 04:40:14,455] {models.py:1335} INFO - Dependencies all met for <TaskInstance: naveed_test.run_this_first 2018-11-10T12:22:44.367293+00:00 [queued]>
[2018-11-10 04:40:14,500] {models.py:1335} INFO - Dependencies all met for <TaskInstance: naveed_test.run_this_first 2018-11-10T12:22:44.367293+00:00 [queued]>
[2018-11-10 04:40:14,501] {models.py:1547} INFO - 
--------------------------------------------------------------------------------
Starting attempt 6 of 6
--------------------------------------------------------------------------------

[2018-11-10 04:40:14,521] {models.py:1569} INFO - Executing <Task(DummyOperator): run_this_first> on 2018-11-10T12:22:44.367293+00:00
[2018-11-10 04:40:14,521] {base_task_runner.py:124} INFO - Running: ['bash', '-c', u'airflow run naveed_test run_this_first 2018-11-10T12:22:44.367293+00:00 --job_id 108 --raw -sd DAGS_FOLDER/test.py --cfg_path /var/folders/zk/29b_jkbx6z58gzz4zv4tzd_40000gn/T/tmphBUWHU']
[2018-11-10 04:40:17,153] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,152] {settings.py:174} INFO - setting.configure_orm(): Using pool settings. pool_size=5, pool_recycle=3600
[2018-11-10 04:40:17,173] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first /Users/naveed/anaconda2/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2018-11-10 04:40:17,173] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first   """)
[2018-11-10 04:40:17,328] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,328] {__init__.py:51} INFO - Using executor LocalExecutor
[2018-11-10 04:40:17,423] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first /Users/naveed/anaconda2/lib/python2.7/site-packages/airflow/bin/cli.py:1595: DeprecationWarning: The celeryd_concurrency option in [celery] has been renamed to worker_concurrency - the old setting has been used, but please update your config.
[2018-11-10 04:40:17,424] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first   default=conf.get('celery', 'worker_concurrency')),
[2018-11-10 04:40:17,485] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,484] {cli.py:464} ERROR - Section ldap Option bind_user does not exist in the config!
[2018-11-10 04:40:17,485] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,484] {cli.py:464} ERROR - Section ldap Option search_scope does not exist in the config!
[2018-11-10 04:40:17,485] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option basedn does not exist in the config!
[2018-11-10 04:40:17,486] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option group_member_attr does not exist in the config!
[2018-11-10 04:40:17,486] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option uri does not exist in the config!
[2018-11-10 04:40:17,486] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option data_profiler_filter does not exist in the config!
[2018-11-10 04:40:17,487] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option bind_password does not exist in the config!
[2018-11-10 04:40:17,487] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option cacert does not exist in the config!
[2018-11-10 04:40:17,487] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option user_filter does not exist in the config!
[2018-11-10 04:40:17,487] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option user_name_attr does not exist in the config!
[2018-11-10 04:40:17,488] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,485] {cli.py:464} ERROR - Section ldap Option superuser_filter does not exist in the config!
[2018-11-10 04:40:17,488] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,486] {cli.py:464} ERROR - Section atlas Option username does not exist in the config!
[2018-11-10 04:40:17,488] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,486] {cli.py:464} ERROR - Section atlas Option host does not exist in the config!
[2018-11-10 04:40:17,488] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,486] {cli.py:464} ERROR - Section atlas Option password does not exist in the config!
[2018-11-10 04:40:17,489] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,486] {cli.py:464} ERROR - Section atlas Option port does not exist in the config!
[2018-11-10 04:40:17,489] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,486] {cli.py:464} ERROR - Section atlas Option sasl_enabled does not exist in the config!
[2018-11-10 04:40:17,489] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,486] {cli.py:464} ERROR - Section kubernetes Option logs_volume_subpath does not exist in the config!
[2018-11-10 04:40:17,489] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option git_repo does not exist in the config!
[2018-11-10 04:40:17,490] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option git_user does not exist in the config!
[2018-11-10 04:40:17,490] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option worker_container_repository does not exist in the config!
[2018-11-10 04:40:17,490] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option image_pull_secrets does not exist in the config!
[2018-11-10 04:40:17,490] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option git_sync_container_tag does not exist in the config!
[2018-11-10 04:40:17,491] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option in_cluster does not exist in the config!
[2018-11-10 04:40:17,491] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option namespace does not exist in the config!
[2018-11-10 04:40:17,491] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option dags_volume_subpath does not exist in the config!
[2018-11-10 04:40:17,492] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,487] {cli.py:464} ERROR - Section kubernetes Option git_sync_init_container_name does not exist in the config!
[2018-11-10 04:40:17,492] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option dags_volume_claim does not exist in the config!
[2018-11-10 04:40:17,492] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option git_subpath does not exist in the config!
[2018-11-10 04:40:17,492] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option worker_service_account_name does not exist in the config!
[2018-11-10 04:40:17,493] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option git_branch does not exist in the config!
[2018-11-10 04:40:17,493] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option logs_volume_claim does not exist in the config!
[2018-11-10 04:40:17,493] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option gcp_service_account_keys does not exist in the config!
[2018-11-10 04:40:17,493] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option airflow_configmap does not exist in the config!
[2018-11-10 04:40:17,493] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option git_sync_container_repository does not exist in the config!
[2018-11-10 04:40:17,494] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option worker_container_tag does not exist in the config!
[2018-11-10 04:40:17,494] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option git_password does not exist in the config!
[2018-11-10 04:40:17,494] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,488] {cli.py:464} ERROR - Section kubernetes Option delete_worker_pods does not exist in the config!
[2018-11-10 04:40:17,494] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,489] {cli.py:464} ERROR - Section lineage Option backend does not exist in the config!
[2018-11-10 04:40:17,495] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,489] {cli.py:464} ERROR - Section hive Option default_hive_mapred_queue does not exist in the config!
[2018-11-10 04:40:17,495] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,489] {cli.py:464} ERROR - Section dask Option cluster_address does not exist in the config!
[2018-11-10 04:40:17,495] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,489] {cli.py:464} ERROR - Section dask Option tls_cert does not exist in the config!
[2018-11-10 04:40:17,495] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,489] {cli.py:464} ERROR - Section dask Option tls_key does not exist in the config!
[2018-11-10 04:40:17,496] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,489] {cli.py:464} ERROR - Section dask Option tls_ca does not exist in the config!
[2018-11-10 04:40:17,496] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,490] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_host does not exist in the config!
[2018-11-10 04:40:17,496] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,490] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_log_id_template does not exist in the config!
[2018-11-10 04:40:17,496] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,490] {cli.py:464} ERROR - Section elasticsearch Option elasticsearch_end_of_log_mark does not exist in the config!
[2018-11-10 04:40:17,497] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,496] {models.py:258} INFO - Filling up the DagBag from /Users/naveed/Mira/hackathon/airflow/dags/test.py
[2018-11-10 04:40:17,571] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,571] {example_kubernetes_operator.py:54} WARNING - Could not import KubernetesPodOperator: No module named kubernetes
[2018-11-10 04:40:17,572] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,571] {example_kubernetes_operator.py:55} WARNING - Install kubernetes dependencies with:     pip install airflow['kubernetes']
[2018-11-10 04:40:17,719] {base_task_runner.py:107} INFO - Job 108: Subtask run_this_first [2018-11-10 04:40:17,719] {cli.py:492} INFO - Running <TaskInstance: naveed_test.run_this_first 2018-11-10T12:22:44.367293+00:00 [running]> on host naveeds-MacBook-Pro.local
[2018-11-10 04:40:19,437] {logging_mixin.py:95} INFO - [2018-11-10 04:40:19,435] {jobs.py:2612} INFO - Task exited with return code 0

